### PR TITLE
resolves #11. Resolved crash on similar token for transport

### LINKED ISSUE #11: Winston transport duplication
Evening, 
I have been testing gateway-admin microservice and found out that tests were failing due to the duplication of winston.transports. 

The exact problem lies here in dialonce/boot/src/winston.js file on line 16.
```
if (token) {
    const logger = winston.add(winston.transports.Logentries, { token }); //here
```

If a logentry with the exact same name is being added, it throws an error an breaks down the code flow.

This test fails: 
```
it('should not allow transport duplicates', () => {
    const notifier = require('../src/bugsnag')();

    const logger = require(modulePath)(notifier, '00000000-0000-0000-0000-000000000000', false);
    require(modulePath)(notifier, '00000000-0000-0000-0000-000000000000', false);

    const mock = sinon.mock(notifier);
    mock.expects('notify').twice().withExactArgs('should notify');

    assert.equal(typeof logger.transports.logentries, 'object');
    assert.equal(typeof logger.transports.console, 'object');

    logger.warn('should notify', { notify: true });
    logger.error('should notify', { notify: true });
    mock.verify();
    mock.restore();
  });
```

### PR DESCRIPTION
**Pull request by Spring3**
Resolving issue with transport token duplication.

### CODE REVIEW

**Comment by Spring3** [comment]
@mrister, @ky23, @jkernech Please check this fix and in case everything is alright, please publish updates to npm so that I could use it further in the next project.

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .PHONY: test
 deps:
 	npm install -g eslint mocha istanbul
-	npm install -g eslint-config-airbnb eslint-plugin-jsx-a11y eslint-plugin-import eslint-plugin-react
+	npm install -g eslint-config-airbnb eslint-plugin-jsx-a11y@2.2.3 eslint-plugin-import eslint-plugin-react
```
Comment (mrister) [inline Makefile]: you should remove these because they are in DevDeps and will be installed via npm install

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .PHONY: test
 deps:
 	npm install -g eslint mocha istanbul
```
Comment (mrister) [inline Makefile]: remove eslint as it is in devDeps

**Comment by mrister** [comment]
@Spring3 feel free to merge away!

**Pull request merged**